src/opengeodeweb_viewer/rpc/viewer/schemas/take_screenshot.json

```diff
@@ -1,5 +1,5 @@
 {
-  "rpc": "take_screenshot",
+  "rpc": "viewer.take_screenshot",
```

Comment:
Should this change be done automatically at generation of all schemas.json?

Reply:
This would change the testing behavior